tests/test_minimal/test_metadata_schema.py

```diff
@@ -1,9 +1,6 @@
-import traceback
```

Comment:
Odd that isort did not catch this; maybe it's only running on `src` files?

Reply:
yeah, maybe